### PR TITLE
test(mpt circuit): transfer eth to non existing address

### PR DESCRIPTION
**Pull request by DreamWuGit**
### Description

[_PR description_]

### Issue Link

[_link issue here_]

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Contents

- [_item_]

### Rationale

[_design decisions and extended information_]

### How Has This Been Tested?

[_explanation_]

<hr>

## How to fill a PR description 

Please give a concise description of your PR.

The target readers could be future developers, reviewers, and auditors. By reading your description, they should easily understand the changes proposed in this pull request.

MUST: Reference the issue to resolve

### Single responsability

Is RECOMMENDED to create single responsibility commits, but not mandatory.

Anyway, you MUST enumerate the changes in a unitary way, e.g.

```
This PR contains:
- Cleanup of xxxx, yyyy
- Changed xxxx to yyyy in order to bla bla
- Added xxxx function to ...
- Refactored ....
```

### Design choices

RECOMMENDED to:
- What types of design choices did you face?
- What decisions you have made?
- Any valuable information that could help reviewers to think critically

### CODE REVIEW

**Comment by lispc** [comment]
at least need to assign the poseidon table inside mpt circuit ...

**Comment by lispc** [comment]
@z2trillion i think we'd better fix here.  So later we can use the TextContext framework to test any bytecode with mpt circuit inside this repo.

File: zkevm-circuits/src/evm_circuit/execution/callop.rs [hunk 1]
```diff
@@ -1561,6 +1567,45 @@ mod test {
...
+        builder
+            .handle_block(&block.eth_block, &block.geth_traces)
+            .unwrap();
+        let mut block = block_convert::<Fr>(&builder.block, &builder.code_db).unwrap();
+        block.mpt_updates.mock_fill_state_roots();
+        let mpt_circuit = MptCircuit::new_from_block(&block);
+        let prover = MockProver::<Fr>::run(12, &mpt_circuit, vec![]).unwrap();
+        println!("prover result: {:?}", prover.verify());
+        assert!(prover.verify().is_ok());
+        //prover.assert_satisfied_par();
```
Comment (lispc) [inline zkevm-circuits/src/evm_circuit/execution/callop.rs]: verify twice

Comment (DreamWuGit) [reply zkevm-circuits/src/evm_circuit/execution/callop.rs]: removed in [1a0e6d8](https://github.com/scroll-tech/zkevm-circuits/pull/949/commits/1a0e6d88d3e972ab4140fe590ad4e14a94780de8) @lispc

**Pull request merged**